### PR TITLE
Changed homepage styling

### PR DESCRIPTION
**Pull request by Mingull**
I changed the styling of the homepage to make it look a bit better and consistent

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,196 +36,218 @@
...
-			<img src="images/dienst_1" />
-		</div>
-		<div class="col-3">
-			<img src="images/dienst_2" />
-		</div>
-		<div class="col-3">
-			<img src="images/dienst_3" />
-		</div>
-	</section>
+		<!-- More images -->
```
Comment (boonsboos) [inline index.html:171]: if you comment out code, why keep it in source control?

Comment (boonsboos) [reply index.html:171]: looked at the conversation in chat, my bad

File: index.html [hunk 2]
```diff
@@ -37,129 +36,142 @@
...
+					<a href="OverOns.html">Over Ons</a>
+					<a href="contact.html">Contact</a>
+				</div>
 			</div>
-		</div>
-	</header>
+		</header>
+
+		<!-- Background banner with title, text and button -->
+		<section class="home-banner flex justify-center align-center">
```
Comment (boonsboos) [inline index.html:48]: on smaller screens  (max ~800px) this element overflows into the next

File: style.css [hunk 3]
```diff
@@ -75,8 +73,9 @@ a {
 
 .home-banner {
 	background-image: url("images/header.jpg");
+	background-size: cover;
 	width: 100%;
-	height: 400px;
+	height: 50vh;
```
Comment (boonsboos) [inline style.css:78]: using vh for the image scales poorly. set the height in the breakpoint instead

File: index.html [hunk 4]
```diff
@@ -19,12 +19,11 @@
 		<link rel="stylesheet" href="style.css" />
 	</head>
 	<body>
-
 		<!-- Header / Navbar -->
 		<header class="flex justify-center">
 			<div
 				id="header-container"
-				class="flex mb-col-12 col-6 justify-center align-center"
+				class="flex mb-col-12 col-6 justify-center align-center py-10"
```
Comment (boonsboos) [inline index.html:26]: if you set the padding here, could you remove the margin set in the #header-container class?

File: index.html [hunk 5]
```diff
@@ -37,129 +36,142 @@
...
+							actie elkaar ontmoeten. Ontdek de nieuwste releases
+							en de klassieke favorieten, van singleplayer
+							avonturen tot multiplayer games voor alle
+							leeftijden.
+						</p>
+						<a href="diensten.html">Lees meer...</a>
+					</div>
+				</div>
+				<div class="mb-col-12 col-4 flex justify-center home-info-box">
+					<img class="mb-col-12 col-12" src="images/dienst_3.jpg" />
```
Comment (boonsboos) [inline index.html:91]: this image is not the same height as the other ones, it looks off

File: index.html [hunk 6]
```diff
@@ -37,129 +36,142 @@
...
-	<section class="usp-section flex justify-center py-15">
-		<div class="mb-col-6 col-3 text-center">
-			<h5>Snelheid & Gemak</h5>
-			<ul class="ups">
-				<li>
-					<img class="invert-color-img py-15" src="images/iconTruck.svg" alt="Vrachtwagen" />
+		<!-- 4 Unique Selling Propositions (USP's) -->
+		<section class="usp-section flex justify-center py-50">
+			<div class="col-8 flex">
+				<div class="mb-col-6 col-3 text-center px-50">
```
Comment (boonsboos) [inline index.html:122]: by setting the padding on this level, the headings are no longer centered
the same can be said for the other ones

**Review by boonsboos — CHANGES_REQUESTED** [review]

File: index.html (hunk 6 above)
Comment (boonsboos) [reply index.html:122]: you could maybe use a gap for this

File: style.css (hunk 3 above)
Comment (Mingull) [reply style.css:78]: I just used what was already there,I just enlarged it a bit

File: index.html (hunk 6 above)
Comment (Mingull) [reply index.html:122]: oh I don't know why i put padding horizontal here.

File: index.html (hunk 5 above)
Comment (Mingull) [reply index.html:91]: they all should be the same aspect-ratio with background-fit/object-fit cover

File: style.css (hunk 3 above)
Comment (boonsboos) [reply style.css:78]: yes, but the size changes if you change the screen

File: index.html (hunk 2 above)
Comment (Mingull) [reply index.html:48]: thats because there are no breakpoints for tablet sizes yet

File: index.html (hunk 6 above)
Comment (boonsboos) [reply index.html:122]: it also results in an overflow on the x axis on screen <=400px wide

File: index.html (hunk 2 above)
Comment (boonsboos) [reply index.html:48]: #16 should fix that

File: index.html (hunk 5 above)
Comment (boonsboos) [reply index.html:91]: image

File: index.html (hunk 2 above)
Comment (Mingull) [reply index.html:48]: #16  needs to be merged with main

File: style.css (hunk 3 above)
Comment (Tippe) [reply style.css:78]: I changed it to pixels in my pull request and fixed the problem. So better to use 400px.

**Review by Tippe — DISMISSED** [review]
beter

File: index.html (hunk 2 above)
Comment (boonsboos) [reply index.html:48]: done

File: index.html (hunk 4 above)
Comment (Mingull) [reply index.html:26]: done

File: index.html (hunk 5 above)
Comment (boonsboos) [reply index.html:89]: seems to due to the text

**Comment by boonsboos** [comment]
alright, merging !

**Review by boonsboos — APPROVED** [review]

**Pull request merged**